eslint.config.cjs

```diff
@@ -91,8 +91,8 @@ module.exports = [
   {
     files: [
       "eslint.config.cjs",
-      "war/postcss.config.js",
-      "war/webpack.config.js",
+      "postcss.config.js",
+      "webpack.config.js",
```

Comment:
Part of the Webpack build invoked from the root Maven module, so the build definition moves to the root Maven module.